sources/Application/Views/ModalDialogs/ImportSampleDialog.cpp

```diff
@@ -76,6 +76,10 @@ void ImportSampleDialog::DrawView() {
 			}
 			if (!current.IsDirectory()) {
 				strcpy(buffer,p.c_str()) ;
+                // temporary UI to show temporary dir file count limit reached
+                if (count == PICO_MAX_FILE_COUNT) {
+                    strcpy(buffer, "[*MAX FILES LIMIT*]");
+                }
```

Comment:
Shouldn't this be independent of the entry being a file or a directory?

Reply:
ah yes of course! 👍🏻